prometheus-beat-exporter.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: prometheus-beat-exporter
+  version: 0.4.0
+  epoch: 1
+  description: Elastic beat-exporter for Prometheus
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - wolfi-baselayout
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/trustpilot/beat-exporter
+      tag: ${{package.version}}
+      expected-commit: 82cf1b275f77c317789852a985243409d0996ceb
+
+  - uses: go/build
+    with:
+      modroot: .
+      packages: .
+      output: beat-exporter
+      ldflags: -w -X github.com/prometheus/common/version.Version=${{package.version}} -X github.com/prometheus/common/version.Revision=$(git rev-parse HEAD) -X github.com/prometheus/common/version.BuildDate=$(date ${SOURCE_DATE_EPOCH:+ -d@${SOURCE_DATE_EPOCH}} "+%Y-%m-%dT%H:%M:%SZ")
+      # CVE-2023-39325 and CVE-2023-44487
+      deps: golang.org/x/net@v0.17.0 github.com/prometheus/client_golang@v1.11
```

Comment:
nit: I think we should probably deprecate this `deps:` option in favour of using `uses: go/bump`, I dont think it's an issue for this as I think our automation will handle it.  Just saying for future packages.

Reply:
let me quickly do that! thanks for the suggestion!